pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -94,6 +95,7 @@ stdenv.mkDerivation rec {
     # Build flags
       "--enable-shared"
       (ifMinVer "0.6" "--enable-pic")
+      (enableFeature (srt != null) "libsrt")
```

This breaks build of `ffmpeg_3`.
```suggestion
      (ifMinVer "4.0" (enableFeature (srt != null) "libsrt"))
```